### PR TITLE
Change alias UI from pop up dialog to use autocomplete panel

### LINKED ISSUE #83: Alias list window pop up is too instrusive
Change UI implementation for showing aliases from window pop up to use auto command suggestion panel to show aliases.

### PR DESCRIPTION
**Pull request by tanboonji**
Closes #83

### CODE REVIEW

File: src/main/java/seedu/address/logic/LogicManager.java [hunk 1]
```diff
@@ -45,6 +45,7 @@
     private final Model model;
     private final Storage storage;
     private final AddressBookParser addressBookParser;
+    private boolean showAlias = false;
```
Comment (oeiyiping) [inline src/main/java/seedu/address/logic/LogicManager.java:48]: Perhaps rename this to better imply that this is a boolean.

**Review by oeiyiping — COMMENTED** [review]
Aside from minor issue mentioned, LGTM w.r.t Model and Commands.

File: src/main/java/seedu/address/logic/LogicManager.java (hunk 1 above)
Comment (tanboonji) [reply src/main/java/seedu/address/logic/LogicManager.java]: Updated the boolean variable to shouldReturnAlias.

**Review by oeiyiping — APPROVED** [review]
LGTM.

**Review by justgnohUG — APPROVED** [review]
LGTM.

File: src/main/java/seedu/address/model/UniqueAliasMap.java [hunk 2]
```diff
@@ -21,9 +24,12 @@
...
 
-    private static final String EMPTY_ALIASES = "You currently have 0 alias";
-
-    private final HashMap<Alias, CommandAlias> aliases = new HashMap<>();
+    private final ObservableMap<Alias, CommandAlias> internalMap = FXCollections.observableMap(new HashMap<>());
+    private final ObservableMap<Alias, CommandAlias> internalUnmodifiableMap =
+            FXCollections.unmodifiableObservableMap(internalMap);
+    private final ObservableList<String> internalList = FXCollections.observableArrayList();
+    private final ObservableList<String> internalUnmodifiableList =
+            FXCollections.unmodifiableObservableList(internalList);
```
Comment (yaowei-soc) [inline src/main/java/seedu/address/model/UniqueAliasMap.java:32]: Any reason why you decided to keep a List and Map instead of converting the map to List?

**Review by yaowei-soc — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/model/UniqueAliasMap.java (hunk 2 above)
Comment (tanboonji) [reply src/main/java/seedu/address/model/UniqueAliasMap.java:32]: This is a temporary solution. I will be looking to change this in the next issue #97 I'm working on.

**Pull request merged**